services/docker-compose.yml

```diff
@@ -453,6 +461,8 @@ volumes:
 networks:
   default:
     attachable: true
+  storage_subnet:
+    attachable: true
```

Comment:
why a special network? is it to isolate datcore adapter?

Reply:
well my reasoning is the following:
- who needs to communicate with datcore-adapter? only storage
--> I go with the least connections. If the other don't need it they don't need to be in the same network. CQFD